src/Static/Html/Ipfs.html

```diff
@@ -3,39 +3,9 @@
 <head>
   <meta charset="utf-8">
   <title>Fission Lobby IPFS Worker</title>
+
+  <meta http-equiv="Refresh" content="0; url='{{{IPFS_WORKER_V1}}}/ipfs.html'" />
```

Comment:
Would this mean that the ipfs iframe in the app changes origin? So, before the origin is `auth.fission.codes` and after it's `ipfs.runfission.com`, right?

This also means that a new IPFS blockstore (indexeddb) is initialised for users at a different origin and they won't re-use their existing blockstore at `auth.fission.codes` anymore.

That might totally be fine?

Reply:
Yeah that's the case, and that's fine.